src/arithmetic/entity_funcs/entity_funcs.c

```
@@ -238,60 +238,79 @@ SIValue AR_OUTGOINGDEGREE
-
-		Map_Get(obj, key, &value);
-		// Return a volatile copy of the value, as it may be heap-allocated.
-		return SI_ShareValue(value);
-	} else if(SI_TYPE(obj) & T_POINT) {
-		// retrieve property key 
-		SIValue key = argv[1];
-		return Point_GetCoordinate(obj, key);
-	} else {
-		// unexpected type SI_TYPE(obj)
-		return SI_NullVal();
+	ASSERT(SIArray_Length(path) == n * 2);
```

cppcheck

[error] Null pointer dereference

(nullPointer)